quinn-proto/src/frame.rs

```diff
@@ -684,6 +691,19 @@ impl Iter {
                 token: self.take_len()?,
             },
             Type::HANDSHAKE_DONE => Frame::HandshakeDone,
+            Type::ACK_FREQUENCY => {
+                let sequence = self.bytes.get_var()?;
+                let ack_eliciting_threshold = self.bytes.get_var()?;
+                let request_max_ack_delay = self.bytes.get_var()?;
+                let reordering_threshold = self.bytes.get_var()?;
```

Comment:
nit: call these inline in the struct expression for concision? Intermediate variables don't seem necessary here.